go/pkg/bertyaccount/service_account.go

```diff
@@ -457,7 +459,7 @@ func (s *service) createAccountMetadata(accountID string, name string) (*Account
 	return meta, nil
 }
 
-func (s *service) ImportAccount(_ context.Context, req *ImportAccount_Request) (*ImportAccount_Reply, error) {
+func (s *service) ImportAccount(context context.Context, req *ImportAccount_Request) (*ImportAccount_Reply, error) {
```

Comment:
```suggestion
func (s *service) ImportAccount(ctx context.Context, req *ImportAccount_Request) (*ImportAccount_Reply, error) {
```

(there are specific rules in go when using `context.Context`)